services/enrichment/fields/locations/src/lib/merge.js

```diff
@@ -0,0 +1,12 @@
+export const mergeRecords = (newRecord, existingRecord) => {
```

Comment:
This file is also duplicated and is a good candidate for making a helper.

Reply:
Actually, now that we only make partial updates, we don't need to merge the whole records anymore. We can focus on merging the field to be enriched. Thanks for noticing this